src/Confluent.Kafka/Impl/SafeConfigHandle.cs

```diff
@@ -103,6 +101,7 @@ internal IntPtr Dup()
 
         internal void Set(string name, string value)
         {
+            CheckClosedHandle();
             // TODO: Constant instead of 512?
```

Comment:
Unrelated to this change, but feel free to implement this TODO in this PR: put a `MaxErrorStringLength` constant (`internal`) in `LibRdKafka` and reference that instead of using 512 explicitly.